model/countries/AU/AU-parsing-rules.yaml

```diff
@@ -0,0 +1,376 @@
+regex_definitions:
+  kBuildingValueRe:  # Regex for type "building"
+    regex_concat:
+      parts:
+      - regex_fragment: \d+  # House number
+      # Covering cases - house number with alphabetic suffix, house number range.
+      - regex_fragment: '(?:'
+      - regex_fragment: \s*[[:alpha:]]\b # Optional suffix for building ids with a letter suffix, e.g. 12a.
+      - regex_fragment: '|'
+      - regex_fragment: \s*?[-]\s*\d+ # Housnumber range (10-17)
+      - regex_fragment: ')?'
+      wrap_non_capture: false
+
+  kUnitTypeLiteralRe:  # Regex for "unit-type"
+    regex_fragment: (?:\b(?:apt\.?|apartment|se\.?|ste\.?|suite|unit|u\.?|room|rm\.?|shop|po box|ms\.?|mail shop|mb\.?|marine berth|vlla\.?|villa|we\.?|warehouse|office|off\.?|fy\.?|factory|sl\.?|stall|site|shed))
+
+  kUnitNameValueRe:  # Regex for "unit-name"
+    regex_fragment: (?:\d+\w?\b|\w\b)
+
+  kFloorLiteralRe:  # Regex for floor literal
+    regex_fragment: (?:\b(?:level|l\.?|floor|fl\.?|basement|b\.?|ground floor|g\.?|lower ground floor|lg\.?|upper ground floor|ug\.?|mezzanine|m\.?))
+
+capture_definitions:
+  StreetAddressDecomposition:
+    capture:
+      output: street-address-alternative-1
+      parts:
+      - capture_reference: ParseInBuildingLocation
+      - separator: {regex_fragment: '.*?'} 
+      - capture_reference: ParseBuildingLocation
+
+  ParseBuildingLocation:
+    capture:
+      output: building-location
+      parts:
+      - capture:
+          output: building
+          parts: [ {regex_reference: kBuildingValueRe} ]
+      - separator: {regex_reference: kWhitespaceSeparator}
+      - capture:
+          output: street
+          parts: [ {regex_reference: kMultipleLazyWordsRe} ]
+
+  ParseInBuildingLocation:
+    capture:
+      output: in-building-location
+      parts:
+        - capture_reference: ParseUnitWithMandatoryPrefix
+        - separator: {regex_reference: kWhitespaceSeparator}
+        - capture_reference: ParseFloorWithMandatoryPrefix
+      quantifier: MATCH_OPTIONAL
+
+  ParseUnitWithMandatoryPrefix:
```

Comment:
I thought that I had left the comment already but I don't see it. `ParseUnitWithMandatoryPrefix` does not fit to `MATCH_OPTIONAL` in the prefix